config/params/testnet_e2e_config.go

```diff
@@ -50,6 +50,7 @@ func E2ETestConfig() *BeaconChainConfig {
 
 	// Prysm constants.
 	e2eConfig.ConfigName = EndToEndName
+	e2eConfig.GenesisValidatorsRoot = [32]byte{75, 54, 61, 185, 78, 40, 97, 32, 215, 110, 185, 5, 52, 15, 221, 78, 84, 191, 233, 240, 107, 243, 63, 246, 207, 90, 210, 127, 81, 27, 254, 149}
```

Comment:
hm, should minimal and e2e configs be hard coded the same root as mainnet?

Reply:
Not sure most of the things seem similar so kept same

Reply:
@Redidacove Since these different configs are copies of mainnet, with overrides explicitly listed only, you don't need to override the mainnet genesis validator root with the same value

Reply:
so I need to remove roots from testnete2e and minimal thats what you are suggesting @prestonvanloon